lib/rubocop/cop/rspec/empty_line_after_final_let.rb

```diff
@@ -28,13 +28,8 @@ def on_block(node)
           latest_let = node.body.child_nodes.select { |child| let?(child) }.last
```

Comment:
"'Add an empty line after the last `let` block." - isn't "block" redundant in this statement?

Reply:
Good point, I’ll fix it. And have the error message reflect whether `let` or `let!` was called.